deselected_tests.yaml

```diff
@@ -726,6 +738,18 @@ gpu:
   - tests/test_common.py::test_f_contiguous_array_estimator[TSNE]
   - manifold/tests/test_t_sne.py::test_tsne_works_with_pandas_output
 
+  # Movement of ensemble algorithms out of preview
```

Comment:
well i guess there worth to have a better explanation of this deselection if we can, because they are deselected because of problems with tests and  we do this to support enabling those algos